todoist.el

```diff
@@ -273,13 +273,13 @@ CACHE is optional param to get projects from cache."
 
 NAME is the name of the project."
   (interactive "sProject name: ")
-  (todoist--query "POST" "/projects" (json-encode `(("name" . ,name)))))
+  (todoist--query "POST" "/projects" (encode-coding-string (json-encode `(("name" . ,name))) 'utf-8)))
```

Comment:
Is there a reason you added `encode-coding-string` to every call rather than modify the `todoist--query` function to perform the encoding?

Reply:
@abrochard 

Thank you for your comment!

As for the reason, because it followed the content proposed with #17 and `json-encode` were called in each function, it corresponded in the same way.

If you think it's better to use DRY, I'll try to fix `encode-coding-string` to be called in the todoist--query function.

Should I move the `json-encode` when I make that change as well? Shall I leave the `json-encode` as it is?

Reply:
Oh that is a good question. From the current specs, the todoist API only accepts JSON so I think it makes sense to move both `encode-coding-string` and `json-encode` to the `todoist--query` function. Thank you bringing this up!